src/main/java/org/jabref/gui/fieldeditors/contextmenu/ProtectedTermsMenu.java

```diff
@@ -45,7 +47,8 @@ public ProtectedTermsMenu(TextArea opener) {
 
     private void updateFiles() {
         externalFiles.getItems().clear();
-        for (ProtectedTermsList list : Globals.protectedTermsLoader.getProtectedTermsLists()) {
+        ProtectedTermsLoader loader = Globals.protectedTermsLoader;
+        for (ProtectedTermsList list : loader.getProtectedTermsLists()) {
```

Comment:
When I get the braces right, it would make sense to directly use a stream to filter the  `! isInternalList ` and then only iterate the result collection and apply the actions on them

Reply:
Well, the loop has side effects (adding to `externalFiles`), so you cannot trivially rephrase that as a lambda. Since it's also not actually the target of the PR, I would prefer not to do this change.

Reply:
That's not what I meant:
You could split that in two parts: First filter all !internal things and return the result as a Collection/List. 
Second step you can  iterate that result collection (a smaller result set)  like it is done now.

Reply:
Ok, if you really want, there you go. Not that this matters, there is a one digit number of lists here ;-)